rbac/management/role/serializer.py

```diff
@@ -57,6 +58,20 @@ class Meta:
         model = ResourceDefinition
         fields = ("attributeFilter",)
 
+    def to_representation(self, instance):
+        """Representation of ResourceDefinitions."""
+        data = super().to_representation(instance)
+        if self._is_workspace_filter(instance):
+            tree_ids = Workspace.objects.descendant_ids_with_parents(instance.attributeFilter.get("value"))
+            data.get("attributeFilter").update({"value": tree_ids})
+        return data
+
+    def _is_workspace_filter(self, instance):
```

Comment:
Seems the instance is resource definition? maybe instance.access.permission.application
and instance.access.permission.resource_type?

Reply:
@astrozzc yeah if you notice in the [role model](https://github.com/RedHatInsights/insights-rbac/pull/1596/files#diff-36e87b5178c904f3090bb466061416ce6ff8fcb98dc15c8a69f407a347312a7fR111-R122) I've added those properties to the resource definition to delegate to, assuming that's what you're referring to?

Reply:
Oh, right! LGTM then